app/scripts/auto-type/index.js

```diff
@@ -185,7 +185,8 @@ const AutoType = {
                 logger.debug('Error during active window check, something is wrong', err);
                 return callback(false);
             }
-            if (activeWindowInfo.id !== windowInfo.id) {
+            this.windowId = windowInfo.id;
```

Comment:
It shouldn't be assigned to `this`, especially in `activeWindowMatches` method, this is an unexpected side-effect. You have access to `windowInfo` in the caller, it should pass it to `runAndHandleResult` directly.

Reply:
Sorry for my bad knowledge of js, I've made another commit to fix this, I hope it's okay now.